test/twofishes/client_test.rb

```diff
@@ -29,6 +29,90 @@ def self.geocode(location)
       assert result.parents
     end
 
+    it 'should accept max keyword argument' do
+      Twofishes::Client.expects(:thrift_client).returns(ZurichWithParents)
+      result = Twofishes::Client.geocode('Zurich', max: 5).first
+      assert result.parents
```

Comment:
This test should test that there are max 5 results, but it tests if parents are included